localstack-core/localstack/services/apigateway/next_gen/execute_api/gateway.py

```diff
@@ -32,15 +32,16 @@ def __init__(self):
                 handlers.method_response_handler,
             ]
         )
-        self.response_handlers.extend(
+        self.exception_handlers.extend(
```

Comment:
this was just a switch around, because exception handlers are executed before response handlers, as we've learned the hard way in the beginning 😄